gui/resources.xml

```diff
@@ -490,14 +503,15 @@ Vocabulary
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Remove the current layer from the layer stack.</property>
           <signal name="activate" handler="remove_layer_cb"/>
         </object>
-        <accelerator key="Delete" modifiers="GDK_SHIFT_MASK"/>
+        <accelerator key="Delete" modifiers="GDK_CONTROL_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="NormalizeLayerMode">
           <property name="label" translatable="yes" context="Menu→Layer (labels), Accel Editor (labels)">Convert to Normal Painting Layer</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Convert the current layer or group to a painting layer in ‘Normal’ mode, preserving its appearance.</property>
           <signal name="activate" handler="normalize_layer_mode_cb"/>
         </object>
+        <accelerator key="n" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
```

Comment:
Yeah, thats why assigned the modifiers to it, however you are right. If the Normalize layer is and advance option, then it's probably better to have the user manually select it. Reverting the change.